spec/Section 4 -- Introspection.md

```diff
@@ -147,6 +147,9 @@ type __Type {
 
   # should be non-null for NON_NULL and LIST only, must be null for the others
   ofType: __Type
+
+  # should be non-null for INPUT_OBJECT only
+  oneField: Boolean
```

Comment:
directive name and introspection should match - `oneOf` here?

Reply:
Probably not. `isDeprecated` and `@deprecated` don't match.

Reply:
Originally I felt that the oneField/oneArgument field names would be more obvious. But since I've taken to calling them `oneof input objects` and `oneof fields` I think using `oneOf` in introspection is fine. Change made.